backend/lucene/src/main/java/org/hibernate/search/backend/lucene/search/projection/impl/LuceneFieldProjection.java

```diff
@@ -42,6 +44,7 @@ public void contributeCollectors(LuceneCollectorsBuilder luceneCollectorBuilder)
 	@Override
 	public void contributeFields(LuceneDocumentStoredFieldVisitorBuilder builder) {
 		codec.contributeStoredFields( absoluteFieldPath, builder::add );
+		codec.contributeNestedDocumentPaths( nestedDocumentPaths, builder::addNestedDocumentPaths );
```

Comment:
`builder.addNestedDocumentPaths( nestedDocumentPaths )` would work just as well.


`codec.contributeStoredFields` exists because there's an actual reason for codec to contribute "low-level" fields, in particular in the case of geopoints. For nested document paths, codecs don't have anything to do, so there's no reason to get them involved. See the implementations of `contributeNestedDocumentPaths`: there's only one, and it's dead simple.

Reply:
makes sense, thanks!